BrokerApiNode/README.md

```diff
@@ -0,0 +1,82 @@
+# Broker APIs
+This example demostrates how to use the Streamr Broker API's from NodeJS. Node version 14/16 or higher is recommended.
+
+## Broker & SDKs
+- [Examples directory](https://github.com/streamr-dev/examples/blob/ECODR-34-broker-and-client/NodeJs/src/broker/)
+
+The Broker acts as a relayer node from it's HTTP/WS/MQTT APIs into the Streamr Network
+
+### Broker setup
+```shell
+$ npm install streamr-broker --global
+$ npm run broker:start
```

Comment:
I believe these instructions install globally, but try to run the package locally? 

I think it should be, 
$ npm install streamr-broker --global
$ streamr-broker-init
$ streamr-broker .streamr/broker-config.json